app/controllers/attendances_controller.rb

```diff
@@ -13,7 +13,7 @@ def destroy
   private
 
     def attendance
-      current_user.attendances.find(params[:id])
+      current_user.student_team.attendances.find(params[:id])
```

Comment:
Ok, thanx for the explanation. So let's just go with your initial approach for now (the one without explicitly raising an exception), since you're right, this controller will very likely change to be orga only in #722 and then we can probably get rid of `student_team` all together.

So let's not waste any more thoughts on this, just go back to the initial version ✌️